### PR TITLE
New section: Rules and judging

### PR DESCRIPTION
**Pull request by carlotacb**

### CODE REVIEW

File: src/data/meals.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const meals: Meals[] = [
   },
   {
     title: 'Saturday Breakfast',
-    mealTitle: 'Cold bocatas and pasteries',
+    mealTitle: 'Cold bocadillo and pasteries',
```
Comment (gerardm27) [inline src/data/meals.ts:70]: bocadillos*

Comment (gerardm27) [inline src/data/meals.ts:70]: Pastries*

File: src/data/meals.ts [hunk 2]
```diff
@@ -160,7 +160,7 @@ export const meals: Meals[] = [
   },
   {
     title: 'Sunday Breakfast',
-    mealTitle: 'Cold bocatas and pasteries',
+    mealTitle: 'Cold bocadillos and pasteries',
```
Comment (gerardm27) [inline src/data/meals.ts:163]: Pastries

**Review by gerardm27 — COMMENTED** [review]
Check comments pls

File: src/data/meals.ts (hunk 1 above)
Comment (carlotacb) [reply src/data/meals.ts:70]: esta oculto

**Review by gerardm27 — APPROVED** [review]
Lgtm

**Pull request merged**